README.md

```diff
@@ -76,26 +76,18 @@ It also provides support for templates:
 <p>Price: {{ foo.price.gross|amount }} ({{ foo.price.net|amount }} + {{ foo.price.tax|amount }} tax)</p>
 ```
 
-**Note:** for template tags to work, you need to add `django_prices` to your `INSTALLED_APPS`.
-
-You can also install the wonderful [`babel`](http://babel.pocoo.org/) library and get proper currency symbols with `prices_i18n`. First install BabelDjango:
-
-```
-$ pip install BabelDjango
-```
-
-Then follow the instruction to add it to your `INSTALLED_APPS` and `MIDDLEWARE_CLASSES`. Finally load the localized template tags:
+**Note:** for template tags to work, you need to add `django_prices` to your `INSTALLED_APPS` and follow `django-babel` [instructions](https://github.com/python-babel/django-babel/#using-the-middleware) and update both your `INSTALLED_APPS` and `MIDDLEWARE_CLASSES`. Finally load the localized template tags:
```

Comment:
That's no longer limited to template tags, we use babel in form widgets.

Reply:
Right!